### ISSUE TITLE
directly support heads-setup using x230-flash

### ISSUE THREAD

**Issue by merge** [merge/skulls#90]
The Heads project includes a "x230-flash" target to support "bootstrapping" on the X230. That's at least some source for confusion for users, and we're not supposed to add to that. Currently:
* The heads "installing" wiki even links to our "external_install_bottom.sh" script, because that can directly be used when flashing "x230-flash" (just another 4M coreboot build) to the top chip.
* We include a "x230_heads.sh" script, that, in turn, is only usable _from_ a running Linux, and for flashing a full Heads ("x230" target) build.

We can think about improving the situation by
* adding a Heads "x230-flash" image directly to our X230 release package, as a 3rd option
* documenting that as part of our guide as "Install Heads / no running Linux required on disk", and:
* remove our "x230_heads.sh" script, because Heads is supposed to be flashed from whithin itself, only.

What would be (legally) required to add? Link to the Heads sources? Document how to build a full Heads image from source.

Or should we _not_ add an image, but only document (and maybe distribute images via an independent "unofficial" project?)?

**Comment by asvc** [comment]
_> What would be (legally) required to add?_ 

Heads is licensed under GPL v2.0 (https://github.com/osresearch/heads/blob/master/COPYING). 
 Some relevant excerpts:

p2(a):
> ...modified files to carry prominent notices stating that you changed the files and the date of any change...

p2(b)
> ...no charge to all third parties..

p2(b)
> If the modified program normally reads commands interactively... print or display an announcement including an appropriate copyright notice...

p3(a)
> Accompany it with the complete corresponding machine-readable source code.

**TL;DR**
If Skulls is derived from GPL project, it shall be distributed in GPL-compliant way.
